src/Driver/Statement.php

```diff
@@ -27,7 +27,7 @@ interface Statement
      *
      * @throws Exception
      */
-    public function bindValue($param, $value, int $type = ParameterType::STRING): void;
+    public function bindValue(string|int $param, mixed $value, int $type = ParameterType::STRING): void;
```

Comment:
Let's declare it as `int|string $param` since `string` is more of an exception than the rule.